opm/parser/eclipse/EclipseState/Tables/TableColumn.cpp

```diff
@@ -22,6 +22,8 @@
 #include <opm/parser/eclipse/EclipseState/Tables/ColumnSchema.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/TableColumn.hpp>
 
+#include <ert/util/ssize_t.h>
```

Comment:
ssise_t is used at several locations in opm-parser, and is able to store -1 in addition to size_t values.

https://github.com/magnesj/opm-parser/blob/72539e28e1c786ba3d1d6b5a3c7192b68a140afb/opm/parser/eclipse/EclipseState/Tables/TableColumn.cpp#L272